schema/Claim.schema.yaml

```diff
@@ -0,0 +1,41 @@
+title: Claim
+'@id': stencila:Claim
+extends: Entity
+role: secondary
+status: unstable
+description: A claim represents specific reviewable facts or statements.
+$comment: |
+  It is usually displayed as a block element and can have a separate title.
+  In the context of a scholarly article this could be a theorem or proof.
+  See also
+  - [Schema.org `Claim`](https://schema.org/Claim)
+  - [JATS `<statement>`](https://jats.nlm.nih.gov/publishing/tag-library/1.2/element/statement.html)
+properties:
+  title:
+    '@id': stencila:title
+    description: Title of the claim.
+    type: string
+  claimType:
+    '@id': stencila:claimType
+    description: Kind of the claim.
+    type: string
+    enum:
+      - Claim
```

Comment:
It feels a little weird / redundant having a `claimType` of `Claim`. I suggest replacing this with `Statement` just so we have an option that encompasses JATS' `<statement>`.

Reply:
Full agreement :-)
Should this be statement in addition to the examples (theorem, proof, ...) given at https://jats.nlm.nih.gov/publishing/tag-library/1.2/element/statement.html

How does this `enum` work, are these just examples? I don't think there will be a definite list. In any case it's good if people use the same terms if possible.

Reply:
> Should this be statement in addition to the examples

In my opinion, yes.

> How does this enum work, are these just examples?

No, data will be validated again the `enum` list. See https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values. It could just be a plain string instead but then as you say there can be deivergence in terms used.

Reply:
Agreed, makes sense that way (and is optional)!